app/utilities/protocol_importers/protocols_io/v3/protocol_normalizer.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+module ProtocolImporters
+  module ProtocolsIO
+    module V3
+      class ProtocolNormalizer < ProtocolImporters::ProtocolNormalizer
+        def load_protocol(id)
+          response = ProtocolImporters::ProtocolsIO::V3::ApiClient.new.single_protocol(id).parsed_response
+
+          normalized_hash = normalize response[:protocol]
+          normalized_hash
+        end
+
+        private
+
+        def normalize(protocol_hash)
+          normalized_data = {}
+          normalized_data[:source_id] = protocol_hash[:id]
+          normalized_data[:name] = protocol_hash[:title]
+          normalized_data[:description] = santz(protocol_hash[:description])
+          normalized_data[:authors] = protocol_hash[:authors].map { |e| e[:name] }.join(', ')
+
+          normalized_data[:steps] = protocol_hash[:steps].map do |e|
+            {
+              source_id: e[:id],
+              name: e[:components].find { |o| o[:type_id] == 6 }[:source][:title],
+              description: santz(e[:components].find { |o| o[:type_id] == 1 }[:source][:description]),
```

Comment:
I am not sure if we want to strip all tags from the description. It contains HTML and right now I believe it's rendered directly into scinote.

Reply:
Right now it is rendered directly, but for normalized data, we should strip html and have our own styling on FE.

Reply:
We discussed this way with @radiokills.